robocop/config.py

```diff
@@ -179,9 +180,9 @@ def preparse(self, args):
 
     def load_args_from_file(self, argfile):
         try:
-            with open(argfile) as arg_f:
+            with FileReader(argfile) as arg_f:
                 args = []
-                for line in arg_f:
+                for line in arg_f.readlines():
```

Comment:
It is not generator so I can only use methods provided in FileReader class - either read() or readlines()